src/views/BaseView.vue

```diff
@@ -186,7 +188,7 @@
   }
 </script>
 
-<style scoped>
+<style scoped override>
```

Comment:
@cesarvh Interesting, I haven't seen this `override` tag before. Where did you find it documented?

Reply:
I forgot where I found it, but I can't seem to find it again. This was back when I was working on the printable view for the degree progress. Removed!